nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css

```diff
@@ -270,11 +272,13 @@ div.modified-attribute-label {
 
 div.modified-attribute-value {
```

Comment:
@scottyaslan please add `font-weight: 500;` and change `padding-top: 5px;` to `padding-top: 2px;`